python/tvm/relay/frontend/onnx.py

```diff
@@ -951,6 +951,20 @@ def from_onnx(self, graph, opset):
                 else:
                     dtype = d_type
                 self._nodes[i_name] = new_var(i_name, shape=tshape, dtype=dtype)
+        # get list of unsupported ops
+        convert_map = _get_convert_map(opset)
+        unsupported_ops = set()
+        for node in graph.node:
+            op_name = node.op_type
+            if op_name not in convert_map and op_name != 'Constant':
```

Comment:
should we check the _identity_list?  even though currently it is empty but it would be good to make logic complete.